packages/idealo.design/scripts/motifuiImporter.js

```diff
@@ -64,7 +62,7 @@ async function extractComponents(subdirectories) {
     return components
 }
 
-async function storeScreenshotFolderName(components) {
+/*export*/ async function storeScreenshotFolderName(components) {
```

Comment:
Dear @rowolff we need to use export to import the functions we would like to test in our tests. Unfortunately we get the following error when using the script with "export" infront of each function:

![image](https://user-images.githubusercontent.com/66468402/125597597-1eaf545e-9e75-42a0-bdef-2bafdd2f1b4d.png)

Reply:
It's what it says in the error message. By default, node uses the old `module.exports` and `require` syntax.

You can either change `export` to `module.exports` (google for the correct syntax) or create a new package.json in the scripts directory, and in this package.json (just for the importer) add `"type" : "module"` 

Then you can use `import` and `export`. Note that you will then also have to change the imports at the top from `require` to `import`. See also: https://stackoverflow.com/questions/63588714/node9374-warning-to-load-an-es-module-set-type-module